### ISSUE TITLE
ua-parser.js check the version of IE in pc does not work?

### ISSUE THREAD

**Issue by 823844596** [faisalman/ua-parser-js#184]
i user ua-parser.js in node js to check the version of ie in pc, and i change the version of ie by developer tool, and i print the log in node.js, the log is same although different version of ie。
here is  my code:
                  var parser = require('ua-parser-js');
                  var ua =parser(req.headers['user-agent'])
                  console.log("UA:",ua);
and the log is :
UA: { ua: 'Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) like Gecko',
   browser: { name: 'IE', version: '11.0', major: '11' },
   engine: { version: '7.0', name: 'Trident' },
   os: { name: 'Windows', version: '7' },
   device: { model: undefined, vendor: undefined, type: undefined },
   cpu: { architecture: 'amd64' } 
   }

can you give me help? thank you very much!

**Comment by faisalman** [comment]
Can you rephrase the problem? I don't see any issue here apart from the same user agent used..
